src/overlays/actors/ovl_Bg_Mori_Elevator/z_bg_mori_elevator.c

```diff
@@ -20,37 +34,252 @@ const ActorInit Bg_Mori_Elevator_InitVars = {
     (ActorFunc)BgMoriElevator_Update,
     NULL,
 };
-*/
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_F32(uncullZoneForward, 2000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 500, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 3000, ICHAIN_CONTINUE),
+    ICHAIN_VEC3F_DIV1000(scale, 1000, ICHAIN_STOP),
+};
+
+extern UNK_TYPE D_060035F8;
+extern Gfx* D_06002AD0;
```

Comment:
```suggestion
extern Gfx D_06002AD0[];
```

This is going to be an array of Gfx because it is an actual display list, not just pointers to them.